Lib/importlib/_bootstrap.py

```diff
@@ -994,7 +994,7 @@ def _gcd_import(name, package=None, level=0):
     return _find_and_load(name, _gcd_import)
 
 
-def _handle_fromlist(module, fromlist, import_):
+def _handle_fromlist(module, fromlist, import_, recursive=False):
```

Comment:
`recursive` should probably be keyword-only, else it's ambiguous what `_handle_fromlist(..., False)` means.